code/modules/admin/artifacts_panel.dm

```diff
@@ -0,0 +1,107 @@
+/datum/admins/proc/artifacts_panel()
+	if (!(SSxenoarch?.initialized))
+		alert("The Xenoarch subsystem hasn't initialized yet!")
```

Comment:
I think you forgot to return here? Same for the next check

Reply:
So no that was actually on purpose, since large artifacts can be spawned by admins/the map even if the xenoarch subsystem hasn't initialized.

Reply:
but now that you mention it I gotta add as second check later, and maybe I could move this alert to the chat instead of a pop-up.

Reply:
>large artifacts can be spawned by admins/the map even if the xenoarch subsystem hasn't initialized

the subsystems don't take too long to initialize, and I don't think it's a good idea to be spawning stuff when light has not even loaded